src/client/components/definition/Definition.jsx

```diff
@@ -9,6 +9,9 @@ const Definition = ({
   <div className='definition'>
     <h2>{ definition.title }</h2>
     <p>{ definition.definition }</p>
+    { definition.equation
```

Comment:
I wonder if it might be more extensible to make a definition have an array of equations.  Then I think you could map over them here and render 0 or 1 or many equations with the same code (no ternary required).  However, this is probably fine for now.